local-cli/runAndroid/runAndroid.js

```diff
@@ -123,18 +123,19 @@ function buildAndRun(args, reject) {
 }
 
 function startServerInNewWindow() {
+  var yargV = require('yargs').argv;
+
   const launchPackagerScript = path.resolve(
     __dirname, '..', '..', 'packager', 'launchPackager.command'
   );
 
   if (process.platform === 'darwin') {
-    child_process.spawnSync('open', [launchPackagerScript]);
+    const termApp = yargV.open ? yargV.open : 'Terminal.app';
+    return child_process.spawnSync('open', ['-a', termApp, launchPackagerScript]);
   } else if (process.platform === 'linux') {
-    child_process.spawn(
-      'xterm',
-      ['-e', 'sh', launchPackagerScript],
-      {detached: true}
-    );
+    const term = yargV.term ? yargV.term : 'xterm';
```

Comment:
Is this `--term`? The PR says the argument is `--open` and the Mac OS branch above uses `--open` too. Should this be `--open` for consistency?